Tasks/DownloadBuildArtifactsV0/task.json

```diff
@@ -256,12 +265,19 @@
         "DownloadingContainerResource": "Downloading items from container resource %s",
         "DefinitionNameMatchFound": "Definition Name %s resolved to id %d",
         "InvalidBuildDefinitionName": "Definition name %s didn't correspond to a valid definition",
-        "UnresolvedDefinitionId": "Could not resolve build definition id"
+        "UnresolvedDefinitionId": "Could not resolve build definition id",
+        "BeginCheckArtifactConsistency": "Starting artifact consistency check",
+        "CorruptedItemsList": "The following files seem to be corrupted during artifact download:",
+        "BuildArtifactNotHealthy": "Downloaded build artifact is not healthy",
+        "BuildArtifactHealthy": "All artifacts items are healthy",
```

Comment:
Can we describe it more precisely? Does this error usually mean some issue during artifact downloading? Could it maybe more clear with 'Artifact items integrity check successfully finished'?

Reply:
Thanks, fixed it!